styles/styles.css

```diff
@@ -0,0 +1,384 @@
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+li {
+  display: inline;
+}
+
+li a {
+  text-decoration: none;
+}
+
+h1 {
+  color: #263E60;
+  margin-left: 2rem;
+}
+
+h2 {
+  color: #263E60;
+}
+
+h3 {
+  color: #263E60;
+}
+
+/* header styling */
+header {
+  background-color: #263E60;
+  margin-bottom: 0px;
+}
+
+.cart {
+  float: right;
+}
+
+.logo {
+  display: block;
+  padding-top: 2rem;
+  margin-right: auto;
+  margin-left: 2rem;
+  padding-bottom: 0px;
+  margin-bottom: 0px;
+}
+
+.slogan {
+  font-style: italic;
+  font-weight: bold;
+  color: #93B255;
+  margin-left: 2rem;
+  margin-top: 0px;
+  padding-top: 0px;
+  padding-bottom: 2rem;
+  margin-bottom: 0px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.cart-icon {
+  padding-top: 1rem;
+  display: inline;
+  margin-bottom: 0px;
+  padding-bottom: 0px;
+}
+
+.checkout-total {
+  display: inline;
+  font-size: .75rem;
+  color: white;
+  font-weight: bold;
+  margin-right: 2rem;
+  margin-bottom: 0px;
+  padding-bottom: 0px;
+}
+
+.checkout-button {
+  background-color: #93B255;
+  font-size: .6rem;
+  border: none;
+  color: white;
+  padding: .5rem 1rem;
+  text-align: center;
+  text-decoration: none;
+  display: block;
+  font-weight: bold;
+  margin-top: .25rem;
+  margin-right: 2rem;
+  cursor: pointer;
+}
+
+/* main nav styling */
+.main-nav {
+  background-color: lightgray;
+  padding-top: .25rem;
+  padding-bottom: .25rem;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+.main-nav li + li:before {
+    content: ' / ';
+    padding: .5rem;
+    color: #263E60;
+    font-weight: bolder;
+}
+
+.main-nav a {
+  color: #263E60;
+  font-weight: bold;
+}
+
+/* footer styling */
+footer {
+  width: 100%;
+  height: 325px;
+  background-color: #3F3F3F;
+  clear: both;
+}
+
+footer p {
+  color: white;
+  text-align: center;
+}
+
+footer a {
+  color: white;
+  font-weight: bold;
+}
+
+footer ul {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  text-align:center;
+}
+
+footer nav li + li:before {
+    content: ' / ';
+    padding: .5rem;
+    color: white;
+    font-weight: bolder;
+}
+
+footer img {
+  display: block;
+  padding-top: 2rem;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+p.copyright {
+  padding-bottom: 2rem;
+}
+
+/* blog styling */
+.blog-entry {
+    margin: 0 5rem 4rem 2rem;
+    overflow: auto;
+}
+.blog-image {
+    float: left;
+}
+.blog-image img {
+    display: block;
+    margin-right: 1rem;
+}
+.blog-content {
+    margin-left: 1rem;
+    margin-top: 0px;
+}
+.blog-title {
+    font-weight: bold;
+    font-size: 125%;
+    margin-top: 0px;
+}
+
+/* dog styling */
+.all-dogs-container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.dog-adoption-card {
+  width: 30%;
+  border: 1px solid gray;
+  margin: 5px;
+}
+
+/* displays 2 cards in a row on table screens with room for margins */
+@media screen and (max-width: 768px) {
```

Comment:
This works well, But I would encourage you to also learn the grid system, it can be a powerful tool.